package.json

```diff
@@ -2,11 +2,13 @@
   "name": "react-onclickoutside",
   "version": "6.0.0",
   "description": "An onClickOutside wrapper for React components",
-  "main": "./index.js",
-  "module": "./index.js",
-  "jsnext:main": "./index.js",
+  "main": "lib/index.js",
```

Comment:
I'd like to keep the es6 version just "index.js". It might get put in a different dir for ES5 purposes but I don't want to put the authoritative file itself in some dir.